elo_system/elo_system.py

```diff
@@ -0,0 +1,29 @@
+from elo_system.tools import SleeperScraper
+
+
+class EloSystem:
+    scraper = None
+    config = dict()
+
+    def __init__(self, config, scraper=None):
+        self.config.update(config)
+        self.league_info = self.config.get("league_info", dict())
+        if scraper:
+            self.scraper = scraper
+        else:
+            self._gen_scraper()
+            self.scraper.login()
+
+    def _gen_scraper(self):
+        ltype = self.league_info.get('ltype', 'sleeper')
+        if ltype == 'sleeper':
```

Comment:
This may certainly be overkill, but I've enjoyed using string enums for things like this if you have to remember a given `ltype` string later. Disregard if not tho.

Reply:
Currently there's only 2 ltypes, won't expand beyond like 10